client/src/components/Footer/index.js

```diff
@@ -1,7 +1,9 @@
+/* eslint-disable max-len */
```

Comment:
I think we are able to get rid of this now?

Reply:
took care of this and the other.